Resources/Private/Build/MediaConsentPlugin/src/main.ts

```diff
@@ -0,0 +1,49 @@
+// Import global styles
+import './scss/style.scss';
+
+// Import assets
+// @ts-ignore
+import jweilandLogo from './assets/jweilandnet.svg';
+
+// Import TypeScript files
+import './ts/mediaConsent.ts';
+
+// Import mock browser data if environment is development
+if (process.env.NODE_ENV !== 'production') {
+  import('./mocks/browser.ts');
+}
+
+// Define root element
+const appRoot = document.querySelector<HTMLDivElement>('#app');
+
+// Create HTML content
+const htmlContent = `
+  <div>
```

Comment:
What is that? HTML in JavaScript. How to overwrite that with other community extensions?

Reply:
Why do we need to overwrite with other community extensions? It's just an example HTML wrap to work on dev.